### PR TITLE
[Enhancement] Lodash to JavaScript conversion 

### LINKED ISSUE #3661: [Proposal] replace lodash methods and other usage to native built-in JavaScript methods

### PR DESCRIPTION
**Pull request by Aigerim-ai**
I have been working on the conversion of lodash for a while. During office hours, I was consulted and I used the plugin [You-Dont-Need-Lodash-Underscore](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) to find relevant conversion files. This pull request (PR) is not intended to be merged, but I need suggestions regarding the folder location where all the conversion methods will be used.

fixes  #3661

### CODE REVIEW

**Comment by abbyhu2000** [comment]
Hi @Aigerim-ai, could you update the PR title to be something descriptive? The PR title should summarize what this PR achieves in a clear way.

**Review by ananzh — CHANGES_REQUESTED** [review]
@Aigerim-ai First of all thank you for the PR. But I think this PR still requires some further modifications before it is ready for a full review.

**Comment by ananzh** [comment]
@Aigerim-ai I would suggest for now revert the PR to a draft state and address some unexpected changes, like updating license, including un-used package, resolving something that is not related to the issue itself and add something unexpected. Then update PR title and CHANGELOG. Rebase it and make it a formal PR. If you need help, let us know.

File: package.json [hunk 1]
```diff
@@ -442,7 +443,7 @@
     "redux": "^4.0.5",
     "regenerate": "^1.4.0",
     "reselect": "^4.0.0",
-    "resize-observer-polyfill": "^1.5.1",
+    "resize-observer-polyfill": "1.5.1",
```
[the remark this answers is not in the document]
Comment (Aigerim-ai) [reply package.json:446]: When I was installing dependency it showed me this issue, `npm ERR! code ETARGET
npm ERR! notarget No matching version found for resize-observer-polyfill@1.5.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of '@elastic/charts'
npm ERR! notarget `

File: src/plugins/opensearch_dashboards_utils/lodash_replacement/lodash_replacement.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function get(obj, path, defaultValue) {
```
[the remark this answers is not in the document]
Comment (Aigerim-ai) [reply src/plugins/opensearch_dashboards_utils/lodash_replacement/lodash_replacement.js:6]: The purpose of this file is to create reusable functions, such that when we will convert from lodash to javascript, we can easily import the functions

File: package.json [hunk 3]
```diff
@@ -379,6 +379,7 @@
     "eslint-plugin-react": "^7.20.3",
     "eslint-plugin-react-hooks": "^4.0.4",
     "eslint-plugin-react-perf": "^3.2.3",
+    "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
```
[the remark this answers is not in the document]
Comment (Aigerim-ai) [reply package.json:382]: Yes, I got you! I am planning to push only changed files from lodash to javascript, the changes made in package.json will not appear in the future reviewed PRs, the plugin I have installed will only be needed to "detect" all lodash functions with in the project.

**Comment by ananzh** [comment]
@Aigerim-ai Due to an extended period without updates, we're going to close this issue for now. We appreciate your contribution and understand that priorities and availability can change. Please feel free to reopen this issue when you have the opportunity to revisit it. Thank you for your understanding.

**Pull request closed without merging**